src/components/DateTime/DateTimePicker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from './DateTimePicker.module.css';
+import classNames from "classnames";
+import { Label } from "../Typography";
+import { FormElement } from "../FormElement";
+
+export type DateTimePickerProps = {
+    label?: string;
+    dateTimeValue?: any;
```

Comment:
Can you also just call this property `value`. With the current name, the usage is `<DateTimePicker dateTimeValue={...}/>`. It’s a little redundant, since you already know it’s the value for the DateTimePicker, seeing as it’s a prop of DateTimePicker.

Reply:
Renamed "dateTimeValue" to "value"